Starcraft2Client.py

```diff
@@ -267,12 +276,7 @@ def build_mission_table(self, dt):
 
                         self.mission_id_to_button = {}
                         categories = {}
-                        available_missions = []
-                        unfinished_locations = initialize_blank_mission_dict(self.ctx.mission_req_table)
-                        unfinished_missions = calc_unfinished_missions(self.ctx.checked_locations,
-                                                                       self.ctx.mission_req_table,
-                                                                       self.ctx, available_missions=available_missions,
-                                                                       unfinished_locations=unfinished_locations)
+                        available_missions, unfinished_missions = calc_unfinished_missions(self.ctx)
```

Comment:
`unfinished_missions` seems to be unused below. You use the `_` placeholder in other places, so this line (or the code below) seems wrong.

Reply:
It is used on line 299